### PR TITLE
Add .env.example

### PR DESCRIPTION
**Pull request by nonameolsson**
This PR adds and example `.env`-file, and some updated instructions in README.md on how to use it.

### CODE REVIEW

File: boilerplate.js [hunk 1]
```diff
@@ -86,7 +86,8 @@ async function install (context) {
     { template: 'ignite.json.ejs', target: 'ignite/ignite.json' },
     { template: '.editorconfig', target: '.editorconfig' },
     { template: '.babelrc', target: '.babelrc' },
-    { template: 'Tests/Setup.js.ejs', target: 'Tests/Setup.js' }
+    { template: 'Tests/Setup.js.ejs', target: 'Tests/Setup.js' },
+    { template: '.env.example', target: '.env.example' },
```
Comment (skellock) [inline boilerplate.js]: Oops.  Trailing comma.

Comment (GantMan) [inline boilerplate.js]: yup, we love trailing comas, but due to StandardJS we don't allow them.

**Comment by nonameolsson** [comment]
@GantMan Oh, thanks! My bad!

**Comment by nonameolsson** [comment]
I would need to update the generated `.gitignore` in project root. Is that something that is possible to do from the boilerplate, or is that done in `ignite-cli`?

**Comment by skellock** [comment]
We would append it to the existing `.gitignore` already in the project (which comes from `react-native-init`).

Add this line to [boilerplate.js](https://github.com/infinitered/ignite-ir-boilerplate/blob/master/boilerplate.js#L109):

```js
filesystem.append('.gitignore', '\n.env\n') // or whatever you're trying to ignore
```

**Comment by jamonholmgren** [comment]
This looks pretty good to me. :shipit: ?

**Comment by nonameolsson** [comment]
@skellock Should I change anything more?

**Comment by GantMan** [comment]
@nonameolsson - one linting bug

/home/runner/ignite-ir-boilerplate/boilerplate.js:118:1: More than 1 blank line not allowed.

**Comment by GantMan** [comment]
@nonameolsson - one merge conflict for ya.  Can you fix so we can merge?   Sorry for the delay.

**Comment by nonameolsson** [comment]
@GantMan What am I missing here?

**Comment by GantMan** [comment]
damn rebase conflicts.  Are you familiar with how to rebase locally to resolve?

**Comment by GantMan** [comment]
Or you can open a second PR with the code in it...


Also, here's Rebase in my opinion:   https://shift.infinite.red/git-time-travel-magic-amend-rebase-319c5a0f4e9c

**Comment by skellock** [comment]
It'd be something like this:

```sh
git remote add upstream git@github.com:infinitered/ignite-ir-boilerplate.git
git fetch origin/master
git checkout envexample
git rebase upstream/master
# then fix the file in conflict
git add the-conflicting-file.js
git rebase --continue
# ^ repeat fix/add/rebase --continue  until git loves you again
git push --force
```

**Comment by nonameolsson** [comment]
@skellock @GantMan Wow, thanks for the help! I hope that I figured it out correctly! A lot of commits...

**Comment by skellock** [comment]
Ha!  I'll squash 'em down here when I merge.  Nobody will ever know the pain you've suffered.

Thanks for the PR! Welcome aboard.

**Pull request merged**